tests/test_fft.py

```diff
@@ -12,9 +11,30 @@ def setUp(self):
     def tearDown(self):
         pass
 
+    def testFft(self):
```

Comment:
I think we list both pyfftw and scipy as dependencies. So given that those two should always work, better to test them both. The way to do that is to override the appropriate `config` option using a context manager (I think the tests for the config should give you an example.)

Reply:
Thanks. Both settings are tested in new commit 0a29638.